testsuite/integration-arquillian/tests/other/server-config-migration/src/test/resources/domain/domain-1.9.8.Final-redhat-1.xml

```diff
@@ -178,6 +178,7 @@
                     <local-cache name="sessions"/>
```

Comment:
Is this XML modification really required? It seems as if we were ensuring the artifact binding works in very old version of KC.

Reply:
Good point. I remove it.